server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -545,6 +613,18 @@ export function configureWebSocketServices(
                 );
                 // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 connection.disconnect();
+                if (isClientConnectivityCountingEnabled && throttleAndUsageStorageManager) {
+                    const connectionTimestamp = connectionTimeMap.get(clientId);
+                    if (connectionTimestamp) {
+                        await storeClientConnectivityTime(
```

Comment:
This `await` in the for loop can/will be problematic. We would have to wait for the storage to occur between notifying the next client of a disconnect. I'd suggest pushing these to a promise array and awaiting outside the loop (see the removeP implementation a few lines down)

Reply:
good point.